target/linux/ipq806x/base-files/etc/board.d/02_network

```diff
@@ -11,6 +11,11 @@ board_config_update
 board=$(board_name)
 
 case "$board" in
+arris,tr4400-v2)
+	ucidef_set_interfaces_lan_wan "eth1" "eth2"
+	ucidef_add_switch "switch0" \
+		"1:lan" "2:lan" "3:lan" "4:lan" "6u@eth1" "0u@eth0"
+	;;
```

Comment:
Is the switch connected to eth0? Could you explain how eth1 and eth2 are used for this device?

Reply:
yes, both eth0 and eth1 are (seem to be) connected to the switch in ports 0 and 6. both can be used for lan. i tested both and chose eth1 for lan because of prior art (a v1 port -not upstreamed- chose that) and because eth1 seems to be commonly used for lan in the devices i've seen.

now here is a quirk: the wan port seems not to be connected to the switch and seems connected to eth2 directly instead. `ip a` shows up/down status according to whether a cable is connected or not to the wan port. however luci shows eth2 as "connected: yes" regardless of actual connection.

besides this detail, the 5 ports work as expected.

Reply:
btw, in the stock fw: eth0 is unused, eth1 is lan, and eth2 is wan. (same in this port.)

Reply:
Okay, that explains. Thanks for the info. Have no idea for the luci.